release sdk package (#33850)
Azure/azure-sdk-for-js: sdk/ai/ai-inference-rest/CHANGELOG.md

# Release History

> ## 1.0.0-beta.7 (2025-04-15)

Reviewer: I didn't see your list has this one?

### Bugs Fixed

> - Add missing apiView and changelog configuration RLC packages (#33575)

Reviewer: ```suggestion
- Add missing apiView and changelog into the package (#33575)
```
